extensions/amp-story/amp-story.md

```diff
@@ -254,53 +250,191 @@ The `bookend-config-src` value is a URL endpoint that returns data for the end s
+The `textbox` component let you specify text inside the bookend. It can be used for the photo credits, for example. It requires a `text` array, where each element of the array is a line of text.
 
-These are configured in the `related-articles` field of the response object.
+```json
+{
+  "type": "textbox",
+  "text": [
+    "Food by Enrique McPizza",
+    "Choreography by Gabriel Filly",
+    "Script by Alan Ecma S.",
+    "Direction by Jon Tarantino"
+  ]
+}
+```
 
 #### Social sharing
 
-The configuration for social sharing is defined in the `share-providers` field of the response object [optional].
+The configuration for social sharing is defined in the `share-providers` field of the response object, and is optional.
 
-This field should contain an object with key-value pairs. Each key represents a share provider's name (e.g. `facebook`). The value should be set to a non-empty configuration object for the provider or `true` (when no parameters are required).
+This field should contain an a string. Each string respresents a share provider's name (e.g. `twitter`).
+
+When extra parameters are required, an object with key-value pairs should be used. The object should contain a key `provider` with a value (e.g. `facebook`) corresponding to the provider's name. The next key-values will depend on the share provider.
 
 The list of available providers is the same as in the [amp-social-share](https://www.ampproject.org/docs/reference/components/amp-social-share) component.
 
-Each of these providers has a different set of available parameters ([see `data-param-*`](https://www.ampproject.org/docs/reference/components/amp-social-share#data-param-*)). The configuration object takes these parameters without the `data-param-` preffix (for example, the `data-param-app_id` would appear in the configuration object as `app_id`).
+Each of these providers has a different set of available parameters ([see `data-param-*`](https://www.ampproject.org/docs/reference/components/amp-social-share#data-param-*)). The configuration object takes these parameters without the `data-param-` prefix (for example, the `data-param-app_id` would appear in the configuration object as `app_id`).
+
+#### JSON configuration
+The `<amp-story-bookend>` must have a `src` attribute pointing to the JSON configuration of the bookend. It is described as a URL endpoint that accepts GET requests and returns a JSON response with the contents of the bookend.  If omitted, the amp-story component renders a default UI for the end screen. The system is responsible for fetching the data necessary to render related and trending articles.  This can be served from a static JSON file, or dynamically-generated (e.g., to calculate what is currently trending).
 
 #### Example JSON response
 
 ```json
 {
+  // It is required to specify you are using the v1.0 version.
```

Comment:
This is actually really important.   Maybe worth figuring out another more prominent place to say it?